pkg/ccl/streamingccl/streamingest/replication_stream_e2e_test.go

```diff
@@ -1143,31 +1143,25 @@ func TestTenantStreamingRetryLoadJob(t *testing.T) {
 	c, cleanup := replicationtestutils.CreateTenantStreamingClusters(ctx, t, args)
 	defer cleanup()
 
+	// Inject an error to fail the resumer.
+	mu.Lock()
+	knobLoadErr = errors.Newf("test error")
+	mu.Unlock()
+
 	producerJobID, ingestionJobID := c.StartStreamReplication(ctx)
 	jobutils.WaitForJobToRun(c.T, c.SrcSysSQL, jobspb.JobID(producerJobID))
 	jobutils.WaitForJobToRun(c.T, c.DestSysSQL, jobspb.JobID(ingestionJobID))
 
 	srcTime := c.SrcCluster.Server(0).Clock().Now()
 	c.WaitUntilReplicatedTime(srcTime, jobspb.JobID(ingestionJobID))
 
-	c.SrcSysSQL.Exec(t, fmt.Sprintf("PAUSE JOB %d", producerJobID))
-	jobutils.WaitForJobToPause(t, c.SrcSysSQL, jobspb.JobID(producerJobID))
-
-	// Write a bit more to be verified at the end.
-	c.SrcTenantSQL.Exec(t, "INSERT INTO d.t2 VALUES (3);")
-
-	// Inject an error to fail the resumer.
-	mu.Lock()
-	knobLoadErr = errors.Newf("test error")
-	mu.Unlock()
-
-	// Resume ingestion.
-	c.SrcSysSQL.Exec(t, fmt.Sprintf("RESUME JOB %d", producerJobID))
```

Comment:
hm how did we manage to get rid of the resume if the ingestion Resumer is expected to fail above? Maybe I don't understand this test/

Reply:
Oh this is the producer job, nvm.